docs/components/modeler/desktop-modeler/process-applications/create-process-application.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Are we sure we don't just want to include this in docs/components/modeler/desktop-modeler/process-applications.md?

Reply:
Yes, we could. There aren't many features at the moment so I was trying to have at least two sub-pages. 🙈